src/Monolog/Logger.php

```diff
@@ -137,12 +160,15 @@ class Logger implements LoggerInterface
      * @param string             $name       The logging channel
      * @param HandlerInterface[] $handlers   Optional stack of handlers, the first one in the array is called first, etc.
      * @param callable[]         $processors Optional array of processors
+     * @param string             $level Minimal level to log
      */
-    public function __construct($name, array $handlers = array(), array $processors = array())
+    public function __construct($name, array $handlers = array(), array $processors = array(), $level = 'DEBUG')
```

Comment:
I would prefer having the `self::DEBUG` as the default value for the `$level` parameter with a further modification of the `setLevel` method.

Reply:
It's not comfortable to use integer value in `*.yml` file when you define a logger. You have to always remember mapping `int -> string`. I believe it's better to use level names instead of the level key.

But I agree it's not ok to have hardcoded string in constructor parameter so I've just changed it to `LogLevel::DEBUG` constant from `Psr/Log` package which is a dependency for `monolog` library.